### ISSUE TITLE
[New NIP]: Changes to staking rewards distribution 

### ISSUE THREAD

**Issue by Jake2270** [NeblioTeam/Neblio-Improvement-Proposals#14]
### Proposal Description

Would it be something the team and community would consider, reducing the staking rewards distributed to stakers with a percentage also going to team to help pay for marketing, listing fees for exchanges, developer grants for people building on Neblio etc? Basically what team receive from stakers is used within the ecosystem so is always put back into doing something that benefits all stakeholders of Neblio. 

So how about 1% going towards the team? And 9% to stakers? In the end the funds are always put to good use and you'd have a wallet where all NEBL are received marked on the explorer for transparency. This isn't paying team a salary or paying additional fees, it is for putting funds to use to help build, improve the platform and ecosystem. 

Awaiting feedback from the team and community. Thanks. 

### Proposal Voting Details

To be determined 

### Other Information

Nothing to add

**Comment by NeblioTeam** [comment]
Opening up for community feedback.

**Comment by MarkSamson** [comment]
I would like this idea but for me it is a must to know beforehand how the Neblio's are going to be spend. Maybe even vote for spendings of this wallet.

**Comment by Jake2270** [comment]
> I would like this idea but for me it is a must to know beforehand how the Neblio's are going to be spend. Maybe even vote for spendings of this wallet.

I think it's not that important because it's basically good to go to a community fund where community could decide how the NEBL is spent, maybe could use voting for that too? So the wallet is clearly marked on the explorer and access to the fund, just be on a case by case basis. So marketing campaign, exchange listing fee, etc. Cant really know right now what funds will he spent on. Community will know that there is now a fund that everyone who is staking, is contributing to and will go towards any Neblio improvement related activities. I think its only logical, since we have this process NIPs and Block Voting so also a community funded wallet to pay for some of the stuff we come up with. I mean one of my proposals was about improving wallet UI, well a fund like this with regular contributions so the more people stake, the more it fills up and we can pay for things like. I totally dont mind receiving 9% instead of 10% but interesting to know how the community feel about it.

**Comment by MarkSamson** [comment]
Yeah I think we mean the same @Jake2270. Its important that the community decides what happens with these funds.

**Comment by RyanXNeb** [comment]
> Yeah I think we mean the same @Jake2270. Its important that the community decides what happens with these funds. 

Absolutely agree.

**Comment by NautoTeam** [comment]
I think it would be best if funds were exclusively spent on future proposals so the community has full transparency.   Once the pot gets a little weight you could make a lot of dev work happen imo and really build a lot

**Comment by RyanXNeb** [comment]
> I think it would be best if funds were exclusively spent on future proposals so the community has full transparency.   Once the pot gets a little weight you could make a lot of dev work happen imo and really build a lot

yes it's a good shout. Makes sense. Access to funds being dependent on successful proposal.vote. if proposal put in and funds are available to cover the cost of whatever it was needed for then funds are awarded upon vote end. Plan and breakdown of costs must be clearly stated in the proposal.

**Comment by Shane-Neblio** [comment]
If it's paid in NEBL's, might cause some SEC troubles for the NEBL token?

Plus, paying a tax in NEBLs to fund stuff means that the NEBL donated may dump on us. 
& wouldn't it be easier to just create a donation address & leave the coding alone. This way no code involved at all. Just create an address & ask for donation at that address.
Better yet, Ask for a donation in Stable-coin like USDC or USDT so we don't have to worry about that dumping on us or losing value in the treasury. 
and that way, if it's a different coin like USDC, the SEC probably won't care because it's not tied to NEBL token.

**Comment by Shane-Neblio** [comment]
Read this one instead. Treasury & Burn tackled through increasing transaction base fee. this way there is no need to punish specifically the nodes. this way everyone feels it.  

https://github.com/NeblioTeam/Neblio-Improvement-Proposals/issues/16

**Comment by Crypt0daz** [comment]
I agree with the idea of a community fund, definitely feel the community should be able to vote for how it is spent aswell. 

I'd considered the same kind of idea, where a 3rd party could make a request to the community for percentage of the funds, for an improvement, app, add on, or marketing idea, and a time frame of how long to complete. If the community approves it with a vote, the funds would be locked in a smart contract and held back, till such a time where the community would vote and agree again, that the idea has been completed satisfactorily. Then the funds would be released. If it fails to be completed, or the community are unhappy with the result, then funds are released back into the community wallet. 

Would personally vote to add 1% higher inflation rather than dropping the nodes from 10% as it is a nice number for stakers to calculate their rewards, or companies to know how much Nebl they need to stake to cover their fees. Will comment on your idea separately Shane.
